src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -50,6 +51,33 @@ public function load(string $path, string ...$paths): void
         $this->doLoad(false, $path, $paths);
     }
 
+    /**
+     * Loads one or several .env and the corresponding env.local, env.$env and env.$env.local files if they exist.
+     *
+     * The env.test.local is always ignored because tests should produce the same results for everyone.
```

Comment:
I don't think we should put any restriction and hardcode this. If that's really what the users want, that's on them. I'd remove it.

Reply:
Why is `.env.test.local` mentioned in the following sentence, then ? 😄 

> .env.dev.local, .env.test.local, .env.prod.local... - Local overrides of environment-specific settings.

Reply:
@ogizanagi the comment was wrong 😁. Fixed!